webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-readonly.js

```diff
@@ -86,6 +95,12 @@ export class SakaiRubricReadonly extends RubricsElement {
     e.stopPropagation();
     this.dispatchEvent(new CustomEvent('copy-to-site', { detail: this.rubric.id }));
   }
+
+  pdfRubric(e) {
+
```

Comment:
space

Reply:
This pdfRubric function seems very repetitive. Can't you generalise it and put it in rubrics-utils or something?